tests/core/test_algorithm.py

```diff
@@ -1,16 +1,6 @@
 import pytest
 
-# check if scipy is available for testing
-try:
-    import scipy
-except ImportError:
-    HAVE_SCIPY = False
-else:
-    del scipy
-    HAVE_SCIPY = True
 
-
-@pytest.mark.skipif(not HAVE_SCIPY, reason="test requires SciPy")
```

Comment:
`scipy` is always installed while insalling `glass` because of the following dependency chain - 
```
flt>=2022.7.27->transformcl>=2022.7.28->gaussiancl>=2022.10.21->glass==2024.2.dev47+gaf57508)
```